pkg/arvo/tests/sys/zuse/crypto/secp256k1.hoon

```diff
@@ -116,4 +116,237 @@
+            a409.3822.299f.31d0.082e.fa98.ec4e.6c89
+          0x6cff.5c3b.a86c.69ea.4b73.76f3.1a9b.cb4f.
+            74c1.9760.89b2.d996.3da2.e554.3e17.7769.
+            9617.64b3.aa9b.2ffc.b6ef.947b.6887.a226.
+            e8d7.c93e.00c5.ed0c.1834.ff0d.0c2e.6da6
+          %.n
+      ==
+      :*  0xdff1.d77f.2a67.1c5f.3618.3726.db23.41be.
+            58fe.ae1d.a2de.ced8.4324.0f7b.502b.a659
+          0x243f.6a88.85a3.08d3.1319.8a2e.0370.7344.
+            a409.3822.299f.31d0.082e.fa98.ec4e.6c89
+          0x123d.da83.28af.9c23.a94c.1fee.cfd1.23ba.
+            4fb7.3476.f0d5.94dc.b65c.6425.bd18.6051
+          %.n
+      ==
+      :*  0xdff1.d77f.2a67.1c5f.3618.3726.db23.41be.
+            58fe.ae1d.a2de.ced8.4324.0f7b.502b.a659
+          0x243f.6a88.85a3.08d3.1319.8a2e.0370.7344.
+            a409.3822.299f.31d0.082e.fa98.ec4e.6c89
+          0x1.7615.fbaf.5ae2.8864.013c.0997.42de.
+            adb4.dba8.7f11.ac67.54f9.3780.d5a1.837c.
+            f197
+          %.n
+      ==
+      :*  0xdff1.d77f.2a67.1c5f.3618.3726.db23.41be.
+            58fe.ae1d.a2de.ced8.4324.0f7b.502b.a659
+          0x243f.6a88.85a3.08d3.1319.8a2e.0370.7344.
+            a409.3822.299f.31d0.082e.fa98.ec4e.6c89
+          0x4a29.8dac.ae57.395a.15d0.795d.dbfd.1dcb.
+            564d.a82b.0f26.9bc7.0a74.f822.0429.ba1d.
+            69e8.9b4c.5564.d003.4910.6b84.9778.5dd7.
+            d1d7.13a8.ae82.b32f.a79d.5f7f.c407.d39b
+          %.n
+      ==
+      :*  0xdff1.d77f.2a67.1c5f.3618.3726.db23.41be.
+            58fe.ae1d.a2de.ced8.4324.0f7b.502b.a659
+          0x243f.6a88.85a3.08d3.1319.8a2e.0370.7344.
+            a409.3822.299f.31d0.082e.fa98.ec4e.6c89
+          0xffff.ffff.ffff.ffff.ffff.ffff.ffff.ffff.
+            ffff.ffff.ffff.ffff.ffff.fffe.ffff.fc2f.
+            69e8.9b4c.5564.d003.4910.6b84.9778.5dd7.
+            d1d7.13a8.ae82.b32f.a79d.5f7f.c407.d39b
+          %.n
+      ==
+      :*  0xdff1.d77f.2a67.1c5f.3618.3726.db23.41be.
+            58fe.ae1d.a2de.ced8.4324.0f7b.502b.a659
+          0x243f.6a88.85a3.08d3.1319.8a2e.0370.7344.
+            a409.3822.299f.31d0.082e.fa98.ec4e.6c89
+          0x6cff.5c3b.a86c.69ea.4b73.76f3.1a9b.cb4f.
+            74c1.9760.89b2.d996.3da2.e554.3e17.7769.
+            ffff.ffff.ffff.ffff.ffff.ffff.ffff.fffe.
+            baae.dce6.af48.a03b.bfd2.5e8c.d036.4141
+          %.n
+      ==
+      :*  0xffff.ffff.ffff.ffff.ffff.ffff.ffff.ffff.
+            ffff.ffff.ffff.ffff.ffff.fffe.ffff.fc30
+          0x243f.6a88.85a3.08d3.1319.8a2e.0370.7344.
+            a409.3822.299f.31d0.082e.fa98.ec4e.6c89
+          0x6cff.5c3b.a86c.69ea.4b73.76f3.1a9b.cb4f.
+            74c1.9760.89b2.d996.3da2.e554.3e17.7769.
+            69e8.9b4c.5564.d003.4910.6b84.9778.5dd7.
+            d1d7.13a8.ae82.b32f.a79d.5f7f.c407.d39b
+          %.n
+      ==
+    ==
+  :_  .
+  ^=  t3
+    %+  turn  cases-pub
+    |=  case-pub
+    ^-  tang
+    %+  expect-eq
+      !>  res
+    !>  (verify:schnorr:ecc pub mes sig)
```

Comment:
In a departure from standard, 3-fixed backstep indentation, test-case vase literals are usually both indented:

```
%+  expect-eq
  !>  expected
  !>  actual
```